runtime/doc/autocmd.txt

```diff
@@ -1116,6 +1116,21 @@ VimResized			After the Vim window was resized, thus 'lines'
 VimResume			After Nvim resumes from |suspend| state.
 							*VimSuspend*
 VimSuspend			Before Nvim enters |suspend| state.
+							*VisualChanged*
+VisualChanged			When Visual selection changes.
+				For `linewise-visual` and `blockwise-visual`
+				the start is the top-left corner and the end
+				is the bot-right corner of the selection.
+				Boundaries are byte-index based.
+				Sets these |v:event| keys:
+				    start_row
+				    start_col
+				    end_row
+				    end_col
```

Comment:
We have `getregionpos()` now, which is the de facto way to represent a "region". Does it make sense to represent the VisualChanged region info in the same way?

Then this would become a single field (see `getregionpos()` for the expected structure).

```suggestion
				    region
```

And we could also provide `v:event.old_region` so that event handlers can check if the region changed or only the cursor moved.